app/styles/components/scheduler.scss

```diff
@@ -14,17 +14,17 @@
   display: none;
 }
 
-.speaker {
-  &::after {
-    content: ', ';
-  }
+// .speaker {
+//   &::after {
+//     content: ', ';
+//   }
 
-  &:last-of-type {
-    &::after {
-      content: '';
-    }
-  }
-}
+//   &:last-of-type {
+//     &::after {
+//       content: '';
+//     }
+//   }
+// }
```

Comment:
We don't comment out code when using git. There is no point keeping dead code in repository. Say the last goodbyes and delete it

Reply:
ok was figuring out that no other place is using this and I was supposed to delete it after I confirmed it.